### ISSUE TITLE
Update adds ref models to model_obj

### ISSUE THREAD

**Issue by hmerwood** [nationalarchives/DiAGRAM#34]
https://github.com/nationalarchives/DiAGRAM/blob/4f0a0699930dc99efa3d628619e71553ab53be1e/R/server.R#L84

**Comment by hmerwood** [comment]
Could put unique(rbind(model_obj$data,..)) but this will only remove duplicates and will add ref models to model_obj

**Comment by hmerwood** [comment]
Doesn't seem to be an issue on the deployed version...

**Comment by jwalton3141** [comment]
Is this still an issue for the version deployed at https://diagram.nationalarchives.gov.uk ?

**Comment by DavidUnderdown** [comment]
I don't think so. We did used to get duplicates if you uploaded a a saved set of models that included the reference models, but I've not noticed that behaviour recently
